.env-devel

```diff
@@ -18,6 +18,12 @@ AGENT_VOLUMES_CLEANUP_S3_PROVIDER=MINIO
 
 API_SERVER_DEV_FEATURES_ENABLED=0
 
+AUTOSCALING_DASK=null
+AUTOSCALING_EC2_ACCESS=null
+AUTOSCALING_EC2_INSTANCES=null
+AUTOSCALING_NODES_MONITORING=null
+AUTOSCALING_POLL_INTERVAL=10
```

Comment:
Are these variables already  in `repo.config`?

Reply:
no, not yet. but coming soon. don't worry I will not forget since that will make sim4life.io fail

Reply:
done now